yt/yt/server/node/data_node/job.cpp

```diff
@@ -540,6 +540,7 @@ class TChunkReplicationJob
         workloadDescriptor.Annotations.push_back(Format("Replication of chunk %v%v",
             ChunkId_,
             isPullReplicationJob ? " as a virtual pull" : ""));
+        workloadDescriptor.DiskFairShareBucketTag = ToString(JobId_);
```

Comment:
Perhaps it is worth explicitly distinguishing master jobs from user jobs?

Reply:
I think they are separated on first level. I see `workloadDescriptor.Category = EWorkloadCategory::SystemReplication;` a couple of lines above.
For user jobs I think it will be `UserBatch`